{"path":"api/CsvHelper/ValidationException","templateID":1,"sharedPropsHashes":{"toc":"Z6psdd"},"localProps":{"className":"/ValidationException","data":"# ValidationException Class\r\n\r\nNamespace: [CsvHelper](/api/CsvHelper)\r\n\r\nRepresents a user supplied validation failure.\r\n\r\n```cs\r\n[System.SerializableAttribute]\r\npublic abstract class ValidationException : CsvHelperException, ISerializable\r\n```\r\n\r\nInheritance Object -> Exception -> CsvHelperException -> ValidationException\r\n\r\n## Constructors\r\n&nbsp; | &nbsp;\r\n- | -\r\nValidationException(ReadingContext) | Initializes a new instance of the ``CsvHelper.ValidationException`` class.\r\nValidationException(ReadingContext, String) | Initializes a new instance of the ``CsvHelper.ValidationException`` class with a specified error message.\r\nValidationException(ReadingContext, String, Exception) | Initializes a new instance of the ``CsvHelper.ValidationException`` class with a specified error message and a reference to the inner exception that is the cause of this exception.\r\n"}}
